packages/core/src/js/colors.ts

```diff
@@ -3,7 +3,6 @@ export const colorsWhite = '#FFFFFF'
 export const colorsBlack = '#000000'
 
 export const colorsBlue = {
-  base: '#0084BD',
```

Comment:
Why the removal of the base colors?

Reply:
Conversation with @sweeting. He uses the scale and references it in the design docs too.